docs/quickstarts/excel-custom-functions-quickstart.md

```diff
@@ -0,0 +1,109 @@
+
+## Build your first custom functions project
+
+To start, you'll use the Yeoman generator to create the custom functions project. This will set up your project with the correct folder structure, source files, and dependencies to begin coding your custom functions.
+
+1. Run the following command and then answer the prompts as follows.
+
+    ```
+    yo office
+    ```
+
+    - Choose a project type: `Excel Custom Functions Add-in project (...)`
+
+    - Choose a script type: `JavaScript`
+
+    - What do you want to name your add-in? `stock-ticker`
+
+    ![Yeoman generator for Office Add-ins prompts for custom functions](../images/12-10-fork-cf-pic.jpg)
+
+    The Yeoman generator will create the project files and install supporting Node components.
+
+2. Go to the project folder.
+
+    ```
+    cd stock-ticker
+    ```
+
+3. Trust the self-signed certificate that is needed to run this project. For detailed instructions for either Windows or Mac, see [Adding Self Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).  
+
+4. Build the project.
+
+    ```
+    npm run build
+    ```
+
+5. Start the local web server, which runs in Node.js.
+
+    - If you'll be using Excel for Windows to test your custom functions, run the following command to start the local web server, launch Excel, and sideload the add-in:
+
+        ```
+         npm run start
+        ```
+        After running this command, your command prompt will show details about starting the web server. Excel will start with your add-in loaded. If you add-in does not load, check that you have completed step 3 properly.
+
+    - If you'll be using Excel Online to test your custom functions, run the following command to start the local web server:
+
+        ```
+        npm run start-web
+        ```
+
+         After running this command, your command prompt will show details about starting the web server. To use your functions, open a new workbook in Excel Online. In this workbook, you'll need to load your add-in. 
+
+        To do this, select the **Insert** tab on the ribbon and select **Office Add-ins**. Next select **Manage My Add-ins > Upload My Add-in**. Browse for your manifest file and upload it. If your add-in does not load, check you've completed step 3 correctly.
+
+## Try out the prebuilt custom functions
+
+The custom functions project that you created by using the Yeoman generator contains some prebuilt custom functions, defined within the **src/customfunctions.js** file. The **manifest.xml** file in the root directory of the project specifies that all custom functions belong to the `CONTOSO` namespace.
+
+In your Excel workbook, try out the `ADD` custom function by completing the following steps in Excel:
+
+1. Within a cell, type `=CONTOSO`. Notice that the autocomplete menu shows the list of all functions in the `CONTOSO` namespace.
```

Comment:
```suggestion
1. Select a cell and type `=CONTOSO`. Notice that the autocomplete menu shows the list of all functions in the `CONTOSO` namespace.
```

Reply:
Random q but why is it that type is ok from an accessibility standards POV? You'd think if we say "select" instead of click everywhere that type similarly suggests having a keyboard structure of some kind (whereas, enter doesn't).